Makefile

```diff
@@ -191,14 +191,14 @@ e2e-test:
 	    -kubeconfig=$(KUBECONFIG) \
 	    -nfd.e2e-config=$(E2E_TEST_CONFIG) \
 	    -nfd.pull-if-not-present=$(E2E_PULL_IF_NOT_PRESENT) \
-	    -ginkgo.focus="\[kubernetes-sigs\]" \
+	    -ginkgo.focus="\[kubernetes-sigs\]NFD master and worker" \
```

Comment:
Don't change this. We want to run also other nfd e2e tests...

Reply:
I always forget to remove this .... we should have this dynamic with variables...  (TODO)